src/referrals/dto/import-referrals.dto.ts

```
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
```

ESLint

[error] Replace `'@nestjs/swagger'` with `"@nestjs/swagger"`

(prettier/prettier)